client/src/containers/widget/map/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { FC } from "react";
+
+import { ComposableMap, Geographies, Geography } from "react-simple-maps";
+
+import { cn } from "@/lib/utils";
+
+import { MapData } from "@/types";
+
+const COLOR_MAP = {
+  1: "hsl(var(--map-1))",
+  2: "hsl(var(--map-2))",
+  3: "hsl(var(--map-3))",
+  4: "hsl(var(--map-4))",
+  5: "hsl(var(--map-5))",
+} as const;
+
+interface MapProps {
+  indicator: string;
+  question: string;
+  data: MapData;
+}
+
+const Map: FC<MapProps> = ({ indicator, question, data }) => {
+  return (
+    <div className="relative h-full">
+      <div className="absolute left-6 top-6 space-y-2">
+        <h3 className="text-base font-semibold">{indicator}</h3>
+        <p className="text-xs text-muted-foreground">{question}</p>
+      </div>
+
+      <ComposableMap
+        style={{ height: "100%" }}
```

Comment:
I think we can add as well `width: 100%` as right now the map is overflowing the container and the map looks slightly different as it should.